app/styles/main.css

```diff
@@ -467,11 +467,12 @@ a.tokenBanner:hover{background-color:#2378CC;}
 .partnerLogo-lux{max-width:70px;}
 .sig-logo-space{margin-top:40px;margin-bottom:30px;}
 
-/* 38. f. - Embed */
-.video-special{display:block;width:640px;height:360px;margin-right:auto;margin-left:auto;}
-.video-special-container{display:block;background-color:#1A1D1F;width:100%;height:100%;max-height:254px;margin-top:50px;padding-top:2px;padding-bottom:2px;text-align:center;}
-div.container-fluid.video-special-container > div > iframe{width:100%;height:250px;}
+/* Tutorials */
```

Comment:
Let's keep in mind that we are moving over to a "Component" based system and keeping the `/* 38. f. - Embed */` comment and it's classes/styles, together will assist in doing so. So let's replace `/* Tutorials */` with `/* 38. f. - Embed */`